orchestra/interface_api/project_management/views.py

```diff
@@ -65,14 +63,9 @@ def reassign_assignment_api(request):
 def revert_task_api(request):
     body = json.loads(request.body.decode())
     try:
-        revert_datetime = datetime.utcfromtimestamp(body['revert_datetime'])
-        revert_datetime = pytz.utc.localize(revert_datetime)
-    except:
-        raise BadRequest('Invalid timestamp provided.')
-
-    try:
-        audit = revert_task_to_datetime(
-            body['task_id'], revert_datetime, body['fake'])
+        audit = revert_task_to_iteration(
```

Comment:
from @jrbotros: "check on awaiting processing" gap in the project managment interface